internal/integratedservices/services/securityscan/spec.go

```diff
@@ -27,7 +27,7 @@ type integratedServiceSpec struct {
 	Policy           policySpec        `json:"policy" mapstructure:"policy"`
 	ReleaseWhiteList []releaseSpec     `json:"releaseWhiteList,omitempty" mapstructure:"releaseWhiteList"`
 	WebhookConfig    webHookConfigSpec `json:"webhookConfig" mapstructure:"webhookConfig"`
-	Registry         *registrySpec     `json:"registry" mapstructure:"registry"`
+	Registries       []*registrySpec   `json:"registry" mapstructure:"registries"`
```

Comment:
We shouldn't break the API. The single registry option should be kept and deprecated. If `registries` is passed to the API, `registry` should be ignored.